arwen/hosttest/execution_test.go

```diff
@@ -983,11 +983,11 @@ func TestExecution_ExecuteOnSameContext_Recursive_Direct(t *testing.T) {
 				Ok().
 				Balance(test.ParentAddress, 1000).
 				BalanceDelta(test.ParentAddress, 0).
-				GasUsed(test.ParentAddress, 25424).
+				GasUsed(test.ParentAddress, 25868).
```

Comment:
why did the gas increase ?

Reply:
Because of C contracts changings. These contracts needed to be changed after the new GetBigIntOrCreate()/GetBigInt() logic was implemented.